test/e2e/tests/releases/utils/methods.ts

```diff
@@ -0,0 +1,211 @@
+}) => {
+  await sanityClient.withConfig(CLIENT_OPTIONS).request({
+    uri: `/data/actions/${dataset}`,
+    method: 'POST',
+    body: {
+      actions: [
+        {
+          actionType: 'sanity.action.release.create',
+          releaseId: releaseId,
+          metadata: metadata,
+        },
+      ],
+    },
+  })
+}
+
+export const archiveRelease = async ({
+  sanityClient,
+  dataset,
+  releaseId,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+}) => {
+  const query = `*[_type == "system.release" && _id == "_.releases.${releaseId}"][0]`
+  const release = await sanityClient.fetch(query)
+
+  if (release.state === 'archived') {
+    return
+  }
+
+  if (release) {
+    sanityClient.withConfig(CLIENT_OPTIONS).request({
+      uri: `/data/actions/${dataset}`,
+      method: 'POST',
+      body: {
+        actions: [
+          {
+            actionType: 'sanity.action.release.archive',
+            releaseId: releaseId,
+          },
+        ],
+      },
+    })
+  }
+}
+
+export const deleteRelease = ({
+  sanityClient,
+  dataset,
+  releaseId,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+}) => {
+  // delete release
+  sanityClient.withConfig(CLIENT_OPTIONS).request({
+    uri: `/data/actions/${dataset}`,
+    method: 'POST',
+    body: {
+      actions: [
+        {
+          actionType: 'sanity.action.release.delete',
+          releaseId: releaseId,
+        },
+      ],
+    },
+  })
+}
+
+export const archiveAndDeleteRelease = async ({
+  sanityClient,
+  dataset,
+  releaseId,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+}) => {
+  await archiveRelease({sanityClient, dataset, releaseId})
+  await waitForReleaseToBeArchived({sanityClient, releaseId})
+  await deleteRelease({sanityClient, dataset, releaseId})
+}
+
+export const discardVersion = ({
+  sanityClient,
+  dataset,
+  versionId,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  versionId: string
+}) => {
+  // discard release
+  sanityClient.withConfig(CLIENT_OPTIONS).request({
+    uri: `/data/actions/${dataset}`,
+    method: 'POST',
+    body: {
+      actions: [
+        {
+          actionType: 'sanity.action.document.discard',
+          draftId: versionId,
+        },
+      ],
+    },
+  })
+}
+
+/**
+ * @returns a random string of letters and numbers (9 characters long)
+ * @internal
+ */
+export function getRandomReleaseId() {
+  return Math.random().toString(36).slice(2)
+}
+
+export const createDocument = (
+  sanityClient: SanityClient,
+  documentMetada: Partial<SanityDocument> & {_id: string; _type: string},
+) => {
+  return sanityClient.withConfig(CLIENT_OPTIONS).create({
+    ...documentMetada,
+  })
+}
+
+export const deleteAllReleases = async ({
+  sanityClient,
+  dataset,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+}) => {
+  const query = `*[_type == "system.release"]`
```

Comment:
```suggestion
  const query = `*[_type == "system.release"]{_id}`
```
(since we only care about the _id anyway)